cafebabel/__init__.py

```diff
@@ -106,6 +108,7 @@ def add_template_helpers():
         return dict(
             get_languages=lambda: app.config.get('LANGUAGES', tuple()),
             get_categories=lambda: app.config.get('CATEGORIES', []),
+            config=app.config,
```

Comment:
What about just passing editors emails if we don't need more? Passing the whole config can be harmful (access to critical keys in debug and so on)

Reply:
Good point. Done.